cms/djangoapps/xblock_config/migrations/0001_squashed_0003_move_course_edit_lti_fields_enabled_flag_model_to_lti_consumer.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.24 on 2021-06-29 09:29
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import opaque_keys.edx.django.models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('xblock_config', '0001_initial'), ('xblock_config', '0002_courseeditltifieldsenabledflag'), ('xblock_config', '0003_move_course_edit_lti_fields_enabled_flag_model_to_lti_consumer')]
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='StudioConfig',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('change_date', models.DateTimeField(auto_now_add=True, verbose_name='Change date')),
+                ('enabled', models.BooleanField(default=False, verbose_name='Enabled')),
+                ('disabled_blocks', models.TextField(default='about course_info static_tab', help_text='Space-separated list of XBlocks on which XBlockAsides should never render in studio')),
+                ('changed_by', models.ForeignKey(editable=False, null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL, verbose_name='Changed by')),
+            ],
+            options={
+                'ordering': ('-change_date',),
+                'abstract': False,
+            },
+        )
```

Comment:
We can simply avoid created the `CourseEditLTIEnabledFlag` model since it's no longer used here, and will be created by `xblock-lti-consumer`.